lib/Backend/FlowGraph.cpp

```diff
@@ -3645,6 +3645,26 @@ Loop::SetLoopTopInstr(IR::LabelInstr * loopTop)
     this->loopTopLabel = loopTop;
 }
 
+bool
+Loop::IsSymAssignedToInSelfOrParents(StackSym * const sym) const
+{
+    if (!this->symsAssignedToInLoop->Test(sym->m_id))
+    {
+        for (Loop* curLoop = this->parent; curLoop != nullptr; curLoop = curLoop->parent)
```

Comment:
Might be a little cleaner to just initialize the loop to `this`

Reply:
Yep, changed.

---
In reply to: [180184073](https://github.com/Microsoft/ChakraCore/pull/4943#discussion_r180184073) [](ancestors = 180184073)